docs/buildah-push.md

```diff
@@ -38,6 +38,16 @@ environment variable. `export REGISTRY_AUTH_FILE=path`
 Use certificates at *path* (\*.crt, \*.cert, \*.key) to connect to the registry.
 The default certificates directory is _/etc/containers/certs.d_.
 
+**--compression-format** *format*
+
+Specifies the compression format to use.  Supported values are: `gzip`, `zstd` and `zstd:chunked`.
+
+**--compression-level** *level*
+
+Tweak the compression level used with the compression
```

Comment:
Suggest
```suggestion
Specify the compression level used with the compression.
```
but please do at least add the period

Reply:
fixed